src/__tests__/integration/passport-bearer/bearer-token-verify.integration.ts

```diff
@@ -0,0 +1,108 @@
+import {IAuthUser} from '../../../types';
+import {VerifyFunction} from '../../../strategies';
+import {Client, createClientForHandler} from '@loopback/testlab';
+import {RestServer} from '@loopback/rest';
+import {Application, Provider} from '@loopback/core';
+import {get} from '@loopback/openapi-v3';
+import {authenticate} from '../../../decorators';
+import {STRATEGY} from '../../../strategy-name.enum';
+import {getApp} from './helpers';
+import {MyAuthenticationSequence} from '../../fixtures/sequences/authentication.sequence';
+import {Strategies} from '../../../strategies/keys';
+
+/**
+ * Testing overall flow of authentication with bearer strategy
+ */
+describe('getting strategy with options', () => {
```

Comment:
can we check and follow the description pattern from loopback repo ?

Reply:
they have not commented on any of their test cases